tests/integration/test_azure_cloud_provider.py

```diff
@@ -105,3 +107,97 @@ async def test_create_persistent_volume(kubernetes, pod_with_volume):
     except asyncio.TimeoutError as e:
         raise AssertionError("Timeout waiting for pod to be ready") from e
     assert pod.status.phase == "Running"
+
+
+@pytest.fixture
+async def loadbalanced_service(kubernetes, ops_test):
+    name = ops_test.model_name
+    deployment = from_dict(
+        dict(
+            kind="Deployment",
+            apiVersion="apps/v1",
+            metadata=dict(name=name, labels=dict(app=name)),
+            spec=dict(
+                selector=dict(matchLabels=dict(app=name)),
+                template=dict(
+                    metadata=dict(labels=dict(app=name)),
+                    spec=dict(
+                        containers=[
+                            dict(
+                                image="gcr.io/google-samples/node-hello:1.0",
+                                name="node-hello",
+                                ports=[dict(containerPort=8080, protocol="TCP")],
```

Comment:
Maybe we could change the image to the current `google-samples/hello-app:1.0`?

Reply:
you know i can!  Just for you!